nixos/modules/services/misc/ersatztv.nix

```diff
@@ -0,0 +1,126 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}:
+
+let
+  inherit (lib)
+    mkIf
+    getExe
+    maintainers
+    mkEnableOption
+    mkOption
+    mkPackageOption
+    ;
+  inherit (lib.types) str path bool;
+  cfg = config.services.ersatztv;
+in
+{
+  options = {
+    services.ersatztv = {
+      enable = mkEnableOption "ErsatzTV";
+
+      package = mkPackageOption pkgs "ersatztv" { };
+
+      user = mkOption {
+        type = str;
+        default = "ersatztv";
+        description = "User account under which ErsatzTV runs.";
+      };
+
+      group = mkOption {
+        type = str;
+        default = "ersatztv";
+        description = "Group under which ErsatzTV runs.";
+      };
+
+      transcodeDir = mkOption {
+        type = path;
+        default = "/var/lib/etv-transcode";
+        description = ''
+          Temporary transcode directory
+        '';
+      };
+
+      configDir = mkOption {
+        type = path;
+        default = "/var/lib/ersatztv";
+        description = ''
+          Directory containing the server configuration files and SQLite Database file
+        '';
+      };
+
+      baseUrl = mkOption {
+        type = str;
+        default = "";
+        description = ''
+          Base URL to support reverse proxies that use paths (e.g. `/ersatztv`)
+        '';
+      };
+
+      openFirewall = mkOption {
+        type = bool;
+        default = false;
+        description = ''
+          Open the default ports in the firewall for the server.
+        '';
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd = {
+      tmpfiles.settings.ersatztvDirs = {
+        "${cfg.transcodeDir}"."d" = {
+          mode = "700";
+          inherit (cfg) user group;
+        };
+        "${cfg.configDir}"."d" = {
+          mode = "700";
+          inherit (cfg) user group;
+        };
+      };
+      services.ersatztv = {
+        description = "ErsatzTV";
+        after = [ "network-online.target" ];
+        wants = [ "network-online.target" ];
+        wantedBy = [ "multi-user.target" ];
+
+        serviceConfig = {
+          Type = "simple";
+          User = cfg.user;
+          Group = cfg.group;
+          UMask = "0077";
```

Comment:
Why discriminate its group?

Reply:
I based this on the Jellyfin service and didn't think to change it when it worked. Would it make more sense to just drop the UMask?